### ISSUE TITLE
Ð°uthorization required.

### ISSUE THREAD

**Issue by da-baranov** [intersystems/ipm#434]
I have IRIS for Health community 2023.3 upgraded over 2023.2. 
Now I cannot install zpm anymore, the process ends up with the following message:
```
#5001: Registry https://pm.community.intersystems.com: Ð°uthorization required.
2023-12-20 14:30:43 0 %ZPM.Installer: Installation failed at 2023-12-20 14:30:43
2023-12-20 14:30:43 0 %Installer: Elapsed time 5.878292s
Time spent 5.980512
```
Tried to install using both recommended methods (by pasting a command to my terminal and by dropping zmp manifest file to the Studio window).

**Comment by evshvarov** [comment]
Hi @da-baranov ! It is a known issue with 2023.3. Please see https://github.com/intersystems/ipm/issues/433 for the temporary workaround

**Comment by da-baranov** [comment]
> It is a known issue with 2023.3 IRIS for Health. Please see #433 for the temporary workaround

Thanks a lot, this helped.
To get rid of the error, do the following:
```
1. Execute the following command 
`do ##class(%ZPM.PackageManager.Developer.Utils).AddGlobalMapping($namespace,"oddStudioDocument:(BEGIN):(""%ZP"")","IRISLIB")
do ##class(%ZPM.PackageManager.Developer.Utils).AddGlobalMapping($namespace,"oddStudioDocument:(""%ZP""):(""A"")","IRISSYS")
do ##class(%ZPM.PackageManager.Developer.Utils).AddGlobalMapping($namespace,"oddStudioDocument:(""Ens""):(""Ent"")","ENSLIB")
```
2. Restart IRIS
3. Switch to the "SYSTEM" namespace and try to run zpm installer again

**Comment by isc-tleavitt** [comment]
This has been resolved in newer IRIS versions (that is, 2024.1), and the workaround is sufficient in the meanwhile.
